modules/python/src2/gen2.py

```diff
@@ -74,6 +74,7 @@
 
 static void pyopencv_${name}_dealloc(PyObject* self)
 {
+    ((pyopencv_${name}_t*)self)->v.${cname}::~${sname}();
```

Comment:
Before there are no explicit destructor.
It is ok and fast for trivial structs, but unacceptable for more complex classes.
As for performance, compiler's optimizations should elimnate possible overhead on trival classes.